### PR TITLE
fix sort_directives violations

### PR DESCRIPTION
**Pull request by pq**
The upcoming linter release includes a fix to `sort_directives` that will trigger a host of new violations.  This fixes them up.

See: https://dart-review.googlesource.com/c/sdk/+/196026

Relevant failures: https://logs.chromium.org/logs/dart/buildbucket/cr-buildbucket.appspot.com/8849409225986491904/+/steps/analyze_flutter_flutter/0/stdout

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by skia-gold** [comment]
Gold has detected about 1 untriaged digest(s) on patchset 1.
View them at https://flutter-gold.skia.org/cl/github/80817

**Comment by jonahwilliams** [comment]
Looks like there were several places where the imports were sorted about the language version comment:

https://github.com/flutter/flutter/pull/80817/files#diff-3c009329e72228284de00a4e71d17f55ccd60a2b47113451df6feb3a54c90728R4-R8

**Comment by pq** [comment]
Ah.  Not good.  Thanks for the catch @jonahwilliams.  I'll tidy up.  Thanks!

**Comment by pq** [comment]
Sorry for the haste.  I should have run `flutter analyze` locally.  Should be clean now. 👍

**Comment by skia-gold** [comment]
Gold has detected about 1 untriaged digest(s) on patchset 4.
View them at https://flutter-gold.skia.org/cl/github/80817

File: packages/flutter/test/widgets/text_golden_test.dart [hunk 1]
```diff
@@ -2,9 +2,9 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter/material.dart';
 @TestOn('!chrome')
```
Comment (goderbauer) [inline packages/flutter/test/widgets/text_golden_test.dart:6]: sa,e

File: packages/flutter_tools/test/commands.shard/permeable/analyze_once_test.dart [hunk 2]
```diff
@@ -4,28 +4,30 @@
```
Comment (goderbauer) [inline packages/flutter_tools/test/commands.shard/permeable/analyze_once_test.dart:4]: There appear to be lots of unrelated format changes in this file?

File: packages/flutter_tools/test/general.shard/ios/ios_device_start_nonprebuilt_test.dart [hunk 3]
```diff
@@ -4,6 +4,8 @@
```
Comment (goderbauer) [inline packages/flutter_tools/test/general.shard/ios/ios_device_start_nonprebuilt_test.dart:4]: Some unrelated format changes in this file as well..

File: packages/flutter_tools/test/general.shard/protocol_discovery_test.dart [hunk 4]
```diff
@@ -4,10 +4,11 @@
```
Comment (goderbauer) [inline packages/flutter_tools/test/general.shard/protocol_discovery_test.dart:4]: unrelated format changes

File: packages/flutter_tools/test/general.shard/resident_devtools_handler_test.dart [hunk 5]
```diff
@@ -4,28 +4,26 @@
```
Comment (goderbauer) [inline packages/flutter_tools/test/general.shard/resident_devtools_handler_test.dart:4]: unrelated format changes

**Review by goderbauer — COMMENTED** [review]
The golden file change is ... unexpected. Maybe it needs to be rebased to latest master?

File: dev/benchmarks/complex_layout/test/measure_scroll_smoothness.dart [hunk 6]
```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:complex_layout/main.dart' as app;
```
[the remark this answers is not in the document]
Comment (pq) [reply dev/benchmarks/complex_layout/test/measure_scroll_smoothness.dart]: Agreed.  Thanks!

**Review by goderbauer — APPROVED** [review]
LGTM
